lib/sidekiq/testing.rb

```diff
@@ -68,15 +68,16 @@ class Client
     def raw_push(payloads)
       if Sidekiq::Testing.fake?
         payloads.each do |job|
-          job['class'].constantize.jobs << Sidekiq.load_json(Sidekiq.dump_json(job))
+          Queues.jobs[job['queue']] << Sidekiq.load_json(Sidekiq.dump_json(job))
         end
         true
       elsif Sidekiq::Testing.inline?
         payloads.each do |job|
-          job['jid'] ||= SecureRandom.hex(12)
           klass = job['class'].constantize
-          klass.jobs.unshift Sidekiq.load_json(Sidekiq.dump_json(job))
-          klass.perform_one
+          worker = klass.new
+          worker.jid = job['jid'] || SecureRandom.hex(12)
```

Comment:
Weird that inline testing was pushing to the queue and then immediately popping it off.  I see we can't call perform_one anymore since we don't push to the queue.